zh/access-dashboard.md

```diff
@@ -0,0 +1,129 @@
+---
+title: 访问 TiDB Dashboard
+summary: 介绍如何在 Kubernetes 环境下访问 TiDB Dashboard
+category: how-to
+---
+
+# TiDB Dashboard 指南
+
+`TiDB Dashboard` 是 TiDB 4.0 专门用来帮助观察与诊断整个 TiDB 集群的可视化面板，你可以在 [TiDB Dashboard](https://github.com/pingcap-incubator/tidb-dashboard) 了解详情。 本篇文章将介绍如何在 Kubernetes 环境下访问 TiDB Dashboard。
+
+## 快速上手
+
+> **注意：**
+>
+> 以下教程仅为演示如何快速访问 `TiDB Dashboard`，请勿在生产环境中直接使用以下方法。 
```

Comment:
No need to enclose "TiDB Dashboard" with ``` ` ```. Please remove all of them in this doc.
```suggestion
> 以下教程仅为演示如何快速访问 TiDB Dashboard，请勿在生产环境中直接使用以下方法。 
```